tools/local-nodescanner/main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"os"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/stackrox/scanner/pkg/analyzer/nodes"
+)
+
+func main() {
+	flag.CommandLine = flag.NewFlagSet(os.Args[0], flag.ExitOnError)
+	fPath := flag.String("fspath", "/host", "Path to the root folder of a filesystem")
+	fRHCOSrequired := flag.Bool("rhcosRequired", true, "Fails scan if filesystem is not RHCOS. Node Scanning only works on RHCOS.")
```

Comment:
`filesystem is not RHCOS` it is actually the OS, not the filesystem. Maybe this could be rephrased?